### PR TITLE
Refreshed README

### PR DESCRIPTION
**Pull request by un-tone**
For https://github.com/Starcounter/Starcounter.Linq/issues/56

@Mackiovello I'd appreciate you take a look at my changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ It is available for downloading as [Starcounter.Linq](https://www.nuget.org/pack
 
 Requires Starcounter version 2.3.1 or later and .NET Framework 4.5.
```
Comment (Mackiovello) [inline README.md]: Does this make sense to have with the added table below?

File: README.md [hunk 2]
```diff
@@ -177,9 +186,58 @@ Objects<Person>().Delete(x => x.Age > 40);
...
+  - `IQueryable.First`
+  - `IQueryable.Single`
+  - `IQueryable.SingleOrDefault`
+- `COUNT` function by using method `IQueryable.Count`
+- `AVG` function by using method `IQueryable.Average`
+- `MIN` function by using method `IQueryable.Min`
+- `MAX` function by using method `IQueryable.Max`
+- `SUM` function by using method `IQueryable.Sum`
+- Calculating expression values in LINQ queries
+- Support First, FirstOrDefault, Single, SingleOrDefault and Any methods - ~#27~
```
Comment (Mackiovello) [inline README.md]: Should these words be marked as code with back-ticks as well?

File: README.md [hunk 3]
```diff
@@ -177,9 +186,58 @@ Objects<Person>().Delete(x => x.Age > 40);
...
+- `DELETE FROM` statement by using methods:
+  - `Starcounter.Linq.Queriable.Delete`
+  - `Starcounter.Linq.Queriable.DeleteAll`
+- Using of generic types for a query context
+- Overrided `IQueryable.ToString` for providing of generated SQL queries
```
Comment (Mackiovello) [inline README.md]: I'm not sure if I understand this sentence. Is `IQueryable.ToString` overridden or can you override it?

File: README.md (hunk 2 above)
Comment (Mackiovello) [reply README.md]: Why do you an issue tag at the end for this one?

File: README.md [hunk 4]
```diff
@@ -177,9 +186,58 @@ Objects<Person>().Delete(x => x.Age > 40);
...
+
+## Well-known unsupported features
```
Comment (Mackiovello) [inline README.md]: I would remove "well-known" and add link to relevant issues below.

File: README.md (hunk 1 above)
Comment (un-tone) [reply README.md]: Agree. Removed.

File: README.md (hunk 2 above)
Comment (un-tone) [reply README.md]: It was left by mistake, thanks for the catch. Removed it.

File: README.md (hunk 3 above)
Comment (un-tone) [reply README.md]: Fixed to "overridden"

File: README.md (hunk 4 above)
Comment (un-tone) [reply README.md]: Removed and added the links.

**Comment by un-tone** [comment]
Cool. Thanks!

**Pull request merged**